src/main/java/org/json/HTTP.java

```diff
@@ -113,36 +113,66 @@ public static JSONObject toJSONObject(String string) throws JSONException {
     public static String toString(JSONObject jo) throws JSONException {
         StringBuilder       sb = new StringBuilder();
         if (jo.has("Status-Code") && jo.has("Reason-Phrase")) {
-            sb.append(jo.getString("HTTP-Version"));
-            sb.append(' ');
-            sb.append(jo.getString("Status-Code"));
-            sb.append(' ');
-            sb.append(jo.getString("Reason-Phrase"));
+            appendResponseHeader(sb, jo);
```

Comment:
Please revert all changes to toString(). Refactoring this code is a judgement call. In this case, I trust Douglas Crockford's decision to include all HTTP string conversions in a single method.